manifests/settings.pp

```diff
@@ -41,4 +42,10 @@
       value => $email_smtp_password,
     }
   }
+
```

Comment:
From what I understand, this condition would check if the setting `authorize_login_delegation_auth_source_user_autocreate` is set to `External`. So would it imply that the user needs to set this setting before running the installer?

Reply:
No, it sets the setting.